src/components/Chat/Conversation.js

```diff
@@ -1,14 +1,57 @@
-import React, { useContext } from 'react'
+import React, { useContext, useEffect, useState } from 'react'
 import Messages from './Messages'
 import Input from './Input'
 import { ChatContext } from '../../context/ChatContext'
+import { AuthContext } from '../../context/AuthContext'
+import { doc, onSnapshot } from 'firebase/firestore'
+import { db } from '../../firebase'
+import sound from '../../sound/notification-sound.mp3'
 
 const Conversation = () => {
   const { data } = useContext(ChatContext)
+  const { currentUser } = useContext(AuthContext)
+  const [userData, setUserData] = useState({})
+  const [count, setCount] = useState(0)
 
-  if (data) {
+  // New Message notification
+  const showNotification = (count) => {
+    const counter = count + 1
+    if (document.hidden && currentUser.uid) {
+      setCount(count + 1)
+      document.title = `(${counter}) Messages`
```

Comment:
Rename `const counter` to newCount, replace setCount(count+1) with setCount(newCount).